omod/src/main/java/org/openmrs/module/aijar/web/controller/AijarResourceController.java

```diff
@@ -0,0 +1,16 @@
+package org.openmrs.module.aijar.web.controller;
+
+import org.openmrs.module.aijar.AijarConstants;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.MainResourceController;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/rest/" + RestConstants.VERSION_1 + "/" + AijarConstants.MODULE_ID)
+public class AijarResourceController extends MainResourceController{
```

Comment:
@reagan-meant rename to UgandaEMRResourceController

Reply:
Done